.travis.yml

```diff
@@ -5,3 +5,4 @@ jdk:
 
 script:
 - ./gradlew clean build test
+- ./gradlew :helios-benchmarks:jmh
```

Comment:
is this going to check the benchmark difference between master and a PR?

Reply:
No, that is planned for the next quarter